app/views/planning_applications/site_notices/new.html.erb

```diff
@@ -38,7 +38,7 @@
       <%= form.govuk_radio_button :required, false, label: { text: "No" }, data: { action: "change->show-hide-form#handleEvent" } %>
     <% end %>
 
-    <div class="govuk-!-display-none" id="site-notice-options">
+    <div id="site-notice-options" class="<%= ' govuk-!-display-none' if @site_notice.errors.blank? %>">
```

Comment:
The form was previously disappearing when there was a validation error. There was a bug where you had to reload the page and choose "Yes" again

Reply:
Use a Rails helper to build this tag rather than injecting ERB into attributes, e.g.

``` erb
<%= tag.div(id: "site-notice-options", class: class_names("govuk-!-display-none": @site_notice.errors.blank?)) do %>
...
<% end %>
```

This lessens the probability of an XSS bug arising